assets/js/phoenix_live_view/dom.js

```diff
@@ -547,6 +547,8 @@ const DOM = {
           if (target.value === sourceValue) {
             // actually set the value attribute to sync it with the value property
             target.setAttribute("value", source.getAttribute(name));
+          } else {
+            logError('Skipped updating value on focused element.', target);
```

Comment:
I think the cleanest way would be to do it like this:

```suggestion
            opts.log && opts.log('Skipped updating value on focused element.', target);
```

and then add a `mergeAttrs` method to `DOMPatch`:

```javascript
mergeAttrs(target, source, opts = {}) {
  DOM.mergeAttrs(target, source, { ...opts, log: this.liveSocket.isDebugEnabled() && console.log })
}
```

Then replace any `DOM.mergeAttrs` calls in DOMPatch with `this.mergeAttrs`.